resources/views/dashboard.blade.php

```diff
@@ -54,6 +44,9 @@
                   //$("#expForm").validate();
                   </script>
               </div>
+              <div>
+                <table class='table'><thead><tr><th>OWED</th><th>OWES</th><th>TTL</th><th>BAL</th></tr></thead><tbody><tr><td class='text-success'><strong>+0</strong></td><td class='text-danger'><strong>-0</strong></td><td>0</td><td>0</td></tr></tbody>
```

Comment:
I don't think its a good user experience if these tables would be push to the bottom when adding an expense. It's better if adding an expense could be done in a popup dialog. But I dont know how hard that could be or that would be feasible at this point of time.